contracts/phoenix_fee_contract/v1/ergoscript/phoenix_v1_hodlcoin_fee.es

```diff
@@ -20,19 +20,20 @@
     // Context Variables: None
 
     // ===== Compile Time Constants ($) ===== //
-    // $devPercentage: (Long, Long)
-    // $phoenixPercentage: (Long, Long)
     // $minerFee: Long
-  
+
     // ===== Context Variables (@) ===== //
     // None
 
+    val devPercentage: (Long, Long) = (60L, 100L)
+    val phoenixPercentage: (Long, Long) = (40L, 100L)
+
```

Comment:
just make feeDenom one value as 100 and change devPercentage to devPercentageNum and phoenixPercentage as phoenixPercentageNum

Reply:
we should talk to them to agree on a value so we only need to do this once

Reply:
yes

Reply:
also maybe make feedenom as 1000 and numerator as 600 and 1000, good to test this actually to make sure values are large enough